rn/Teacher/ios/Teacher/AppDelegate.swift

```diff
@@ -52,6 +52,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         #endif
         if hasFirebase {
             FirebaseApp.configure()
+            let remoteConfig = RemoteConfig.remoteConfig()
+            remoteConfig.activate { error in
+                guard error == nil else {
+                    return
+                }
+                let keys = remoteConfig.allKeys(from: RemoteConfigSource.remote)
+                for key in keys {
+                    UserDefaults.standard.set(remoteConfig.configValue(forKey: key).boolValue, forKey: ExperimentalFeature.settingsKey(forConfigKey: key))
+                }
+            }
+            remoteConfig.fetch(completionHandler: nil)
```

Comment:
What we are doing here is saying that every time the app starts, fetch the remote config. This will only reach out to firebase if default duration of 12 hours (I believe) has elapsed since the last time it reached out to firebase. Once the remote config is fetched, it will store the new remote config on disk but not make it active. Then the next time a user opens the app it will call `activate` which will look to see if there is a remote config on disk waiting to be activated. If so it will do it and then run the completion handler. This completion handler is setting the values of the config to UserDefaults so that the `ExperimentalFeature`s can pick up on the values.